packages/gatsby-theme-carbon/src/components/ResourceCard/ResourceCard.js

```diff
@@ -90,7 +90,11 @@ export default class ResourceCard extends React.Component {
       );
     } else {
       cardContainer = (
-        <a href={href} className={carbonTileclassNames} {...rest}>
+        <a
+          href={rest.download ? withPrefix(href) : href}
```

Comment:
Hey! Thanks for the PR. This won't work if the download is from a different site, it will apply the path prefix for the current page.

Reply:
Good point Vince. Maybe testing to see if an http protocol is present (or if it is being defaulted) at the beginning of the string would be a sufficient check on whether the resource is internal or external.

```
var href = 'https://1.dam.s81c.com/m/7e6c4e33b202794c/original/06222020-ls-watson-works-2304x983.jpg'
RegExp(/^(https?:)?\/\//g).test(href)  //true

href = 'http://1.dam.s81c.com/m/7e6c4e33b202794c/original/06222020-ls-watson-works-2304x983.jpg'
RegExp(/^(https?:)?\/\//g).test(href)  //true

href = '//1.dam.s81c.com/m/7e6c4e33b202794c/original/06222020-ls-watson-works-2304x983.jpg'
RegExp(/^(https?:)?\/\//g).test(href)  //true

href = '/1.dam.s81c.com/m/7e6c4e33b202794c/original/06222020-ls-watson-works-2304x983.jpg'
RegExp(/^(https?:)?\/\//g).test(href)  //false

href = '/http://1.dam.s81c.com/m/7e6c4e33b202794c/original/06222020-ls-watson-http://works-2304x983.jpg'
RegExp(/^(https?:)?\/\//g).test(href)  //false


```

Reply:
The code below seems to handle the example I posted previously. However, Chrome and Firefox (and probably most browsers) only support same origin download links (https://caniuse.com/#feat=download). Testing the above url for the image showed that Chrome loads the image in current browser tab, which could be changed to open in a new tab by supplying `target="_blank"`.

```
    let cardContainer;
    if (disabled === true || href === undefined) {
      cardContainer = <div className={carbonTileclassNames}>{cardContent}</div>;
    } else if (isLink === true) {
      cardContainer = (
        <Link to={href} className={carbonTileclassNames} {...rest}>
          {cardContent}
        </Link>
      );
    } else {
      cardContainer = (
        <a
          href={
            rest.download && !RegExp(/^(https?:)?\/\//g).test(href)
              ? withPrefix(href)
              : href
          }
          className={carbonTileclassNames}
          {...rest}
        >
          {cardContent}
        </a>
      );
    }
```